user-guide/Troubleshooting/Procedures/Cassandra_expired_blocking_SSTables.md

```diff
@@ -0,0 +1,49 @@
+---
+uid: Cassandra_expired_blocking_SSTables
+---
+
+# Expired SSTables blocking other tables from being dropped
+
+In case the only data SSTables in Cassandra contain is tombstones, and the tables are therefore expired, they could block other tables that are also expired from being dropped.
+
+To resolve this issue, you can enable the *unsafe_aggressive_sstable_expiration* option for the Cassandra database. This is something that should only be done to troubleshoot this issue, because it is considered a risky option otherwise. The data in the expired SSTables will be dropped without being checked, but as it consists entirely of tombstone data in this case, this is not a problem.
+
+To enable this option, follow the steps below.
```

Comment:
For which kind of setup is this procedure intended? For Cassandra clusters hosted on Linux?

Reply:
I would expect this tag can be enabled for all Cassandra based systems. Anyhow, this procedure has been tested and tried for Linux systems, and as there is no clear documentation on it online, I would limit it to that.